intercom-plugin/src/android/intercom.gradle

```diff
@@ -24,6 +24,7 @@ repositories {
     jcenter()
     maven { url 'https://maven.google.com' }
 }
+
```

Comment:
💥

Reply:
Deliberate 😁 I like the spacing after this block

Reply:
💅